r/_pkgdown.yml

```diff
@@ -60,10 +60,6 @@ template:
     image:
       src: https://arrow.apache.org/img/arrow-logo_horizontal_black-txt_white-bg.png
       alt: "Apache Arrow logo, displaying the triple chevron image adjacent to the text"
-    twitter:
-      creator: "@apachearrow"
-      site: "@apachearrow"
-      card: summary_large_image
```

Comment:
@thisisnic it seems that Open Graph protocol doesn't support BlueSky metadata (only Twitter and possibly Facebook) so I removed the twitter card. Can you approve if this is correct?

Reply:
Makes sense!